modules/core/location/farm_location.base_fields.inc

```diff
@@ -42,8 +42,14 @@ function farm_location_asset_base_fields() {
     'label' => t('Current geometry'),
     'computed' => AssetGeometryItemList::class,
     'hidden' => 'form',
-    'weight' => [
-      'view' => 95,
+    'view_display_options' => [
+      'label' => 'hidden',
+      'type' => 'farm_map_geofield',
+      'settings' => [
+        'output_format' => 'wkt',
+        'output_escape' => TRUE,
+      ],
+      'weight' => 95,
```

Comment:
Instead of explicitly setting `view_display_options` here (which requires setting all the other bits and sub-settings), can we leave this as it was, but put a line after the call to `\Drupal::service('farm_field.factory')->baseFieldDefinition($options);` which simply sets the `label` to `hidden`?

For example:

```
$fields['geometry'] = \Drupal::service('farm_field.factory')->baseFieldDefinition($options);

// Hide the geometry field label in the view display.
$view_display_options = $fields['geometry']->getDisplayOptions('view');
$view_display_options['label'] = 'hidden';
$fields['geometry']->setDisplayOptions('view', $view_display_options);
```

This makes it very explicit what we're doing, and means that if any of those other settings change in `farm.field_factory` in the future we don't need to remember to update them here as well.